vsts/pipelines/nightly.yml

```diff
@@ -55,6 +55,7 @@ jobs:
     - Job_RuntimeImages
   pool:
     name: OryxLinux
+  timeoutInMinutes: 150
```

Comment:
added this to provide more time to the node-sass integration tests (+37 minutes)